dashboard/modules/job/job_manager.py

```diff
@@ -67,7 +83,10 @@ def __init__(self, job_id: str, metadata: Dict[str, str]):
         self._runtime_env = ray.get_runtime_context().runtime_env
 
         self._metadata = metadata
-        self._metadata[JOB_ID_METADATA_KEY] = job_id
+        if JOB_ID_METADATA_KEY not in self._metadata:
```

Comment:
I think a better pattern here may be to use dict update notation + a default metadata dict. so it's obvious what's going on
```
self._metadata = {
** default_metadata,
** passed_metadata
}
```

Reply:
done